plugins/BEdita/Core/tests/TestCase/Model/Table/RolesTableTest.php

```diff
@@ -155,4 +156,23 @@ public function testFindMine()
 
         static::assertEquals($expected, $result);
     }
+
+    /**
+     * Test delete admin role
+     *
+     * @return void
+     *
+     * @covers ::beforeDelete
+     */
+    public function testDeleteAdminRole()
+    {
+        $roles = $this->Roles->get(\BEdita\Core\Model\Table\RolesTable::ADMIN_ROLE);
```

Comment:
For namespaced classes we usually `use` them at the top of the file to avoid having long class names in the code. `use`s should be in alphabetical order.

For classes in the topmost namespace (i.e. `\Exception`) we accept both solutions, if I remember well.